test/io.jl

```diff
@@ -17,30 +17,26 @@ config = Wflow.Config(tomlpath)
     @test dirname(config) == dirname(tomlpath)
 
     # test if the values are parsed as expected
-    @test config.starttime === DateTime(2000, 1, 1)
-    @test config.endtime === DateTime(2000, 2)
+    @test config.time.starttime === DateTime(2000, 1, 1)
+    @test config.time.endtime === DateTime(2000, 2)
     @test config.output.path == "output_moselle.nc"
     @test config.output isa Wflow.Config
-    @test collect(keys(config.output)) == ["routing", "land", "path"]
-
-    # theta_s can also be provided under the alias theta_s
-    @test Wflow.get_alias(
-        config.input.land.soil.parameters,
-        "theta_s",
-        "theta_s",
-        nothing,
-    ) == "thetaS"
-    val = pop!(config.input.land.soil.parameters, "theta_s")
-    config.input.land.soil.parameters["theta_s"] = val
-    @test Wflow.get_alias(
-        config.input.land.soil.parameters,
-        "theta_s",
-        "theta_s",
-        nothing,
-    ) == "thetaS"
+    @test collect(keys(config.output)) == ["variables", "path"]
+
+    # test if soil_water__saturated_volume_fraction can also be provided under the alias theta_s
+    lens = @optic(_.input.parameters.soil_water__saturated_volume_fraction)
+    lens_alias = @optic(_.input.parameters.theta_s)
+    @test Wflow.get_alias(config, lens, lens_alias, nothing) == "thetaS"
+    val = pop!(config.input.parameters, "soil_water__saturated_volume_fraction")
+    config.input.parameters["theta_s"] = val
+    @test Wflow.get_alias(config, lens, lens_alias, nothing) == "thetaS" == "thetaS"
```

Comment:
This last part is not necessary right?

```suggestion
    @test Wflow.get_alias(config, lens, lens_alias, nothing) == "thetaS"
```